scripts/gis-material-update/automation/automation.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/sh
+
+for tormays_source in $*
+do
+    # Fetch data
+    echo "Fetching $tormays_source ..."
+    case $tormays_source in
+    hsl)
+        sh /haitaton-gis/fetch_all.sh $tormays_source hki
+        ;;
+    tram_infra)
+        sh /haitaton-gis/fetch_all.sh hki osm helsinki_osm_lines
```

Comment:
Looks like the `osm_vrt_clip.vrt` file is not copied to the docker image. 
```
❯ docker run --rm --network host --env-file .env haitaton-gis-automation tram_infra
Fetching tram_infra ...
Warning 6: Progress turned off as fast feature count is not available.
0...10...20...30...40...50...60...70...80...90...100 - done.
FAILURE:
Unable to open datasource `/haitaton-gis/osm_vrt_clip.vrt' with the following drivers.
  -> `FITS'
...
```

Reply:
Added osm_vrt_clip.vrt file to docker